Doc/library/sqlite3.rst

```diff
@@ -1098,14 +1098,18 @@ Blob Objects
 Exceptions
 ----------
 
+The exception hierarchy is defined by DB-API 2.0.
+
 .. exception:: Warning
 
-   A subclass of :exc:`Exception`.
+   This exception is not raised by :mod:`sqlite3`.
```

Comment:
Wait, why is it documented here if it isn't raised by the module? Also, redundant link:

```suggestion
   This exception is not raised by ``sqlite3``.
```

Reply:
> Wait, why is it documented here if it isn't raised by the module?

It's part of the API the sqlite3 module expose (via the PEP 249 DB API), so I think it is better to document it, and state explicitly that it's not (currently) used :)

Reply:
Hm, yeah, that's a possibility. This is what the DB API says:

> Exception raised for important warnings like data truncations while inserting, etc.

Reply:
Anyway, I'll defer to your judgement on this one, as the sqlite expert 👍

Reply:
Nice!

Reply:
Nice!